larky/src/main/resources/vendor/Crypto/Hash/RIPEMD160.star

```diff
@@ -15,20 +28,41 @@ def RIPEMD160Hash(object):
         :vartype digest_size: integer
     
     """
-    def __init__(self, data=None):
+    self = larky.mutablestruct(__class__='RIPEMD160Hash')
```

Comment:
this should be: `larky.mutablestruct(__name__='RIPEMD160Hash', __class__=RIPEMD160Hash)`